frontend/src/metabase/admin/settings/components/CloudPanel/CloudPanel.tsx

```diff
@@ -68,21 +68,15 @@ export const CloudPanel = ({
     [dispatch, migrationState],
   );
 
-  const storeUrl = useSetting("store-url");
-
-  const checkoutUrl = useMemo(() => {
-    return migration
-      ? `${storeUrl}/checkout?migration-id=${migration.external_id}`
-      : `${storeUrl}/checkout`;
-  }, [migration, storeUrl]);
-
   const [createCloudMigration, createCloudMigrationResult] =
     useCreateCloudMigrationMutation();
 
+  const checkoutUrl = useStoreUrl("checkout");
```

Comment:
Isn't this now going to default always to `${storeUrl}/checkout`?
Seems like the checkout url is going to be wrong for these two lines:
- https://github.com/metabase/metabase/pull/63771/files#diff-614ef5f440de0a9780da8332845a491165d1894acf09a58018486aafbd8ddd12R94
- https://github.com/metabase/metabase/pull/63771/files#diff-614ef5f440de0a9780da8332845a491165d1894acf09a58018486aafbd8ddd12R103

...or I'm misreading something?

Reply:
Those diff links don't work for me 😕

`checkoutUrl` will be be `${storeUrl}/checkout` but later in `getMigrationUrl` we're adding the `migration-id` query parameter.

In all 3 places were the URL is used  (`handleCreateMigration`, `MigrationInProgress`, `MigrationSuccess`) we always have access to existing `migration`, we will always link to the Checkout with `migration-id` query parameter.

Before, the conditional was there only because at that code flow place, migration can be `undefined`.